### DISCUSSION TITLE
Verbosity may be less significant in multi-language texts

### DISCUSSION THREAD

**Discussion by cdrigby** [syvwlch/Data-Ignota#37] [General]
Tangential but perhaps of interest:
With reference to https://syvwlch.github.io/Data-Ignota/viz.html

Most of the text is in English, so the non-English passages won't effect the results much. However, for a multi-language text, simply counting the number of words might affect the analysis.

Spoken human languages, despite all there differences, may all transmit information at the same rate.
(N. B., EFF Privacy Badger blocks six active trackers)

https://www.science.org/content/article/human-speech-may-have-universal-transmission-rate-39-bits-second

 As a trivial example, it takes two words in English to say what one word can in German, for the radiation produced when an electron decelerates in an electric field, typically that of an atomic nucleus.

English: Braking radiation
German: Bremsstrahlung (https://en.wikipedia.org/wiki/Bremsstrahlung)

One of the marvels of linguistics to me was my first Russian textbook, where the Russian text was about 2/3 the size of the English translation. Russian has six full cases, three genders, and elides the verb "to be" in the present tense, as best I remember. It simply conveys more information/word.

**Comment by syvwlch** [comment]
Very true!

However in this case the language indicated is _before_ translation to English, and the word count is _after_, so we are comparing word count in English. 

Mycroft translates most of the dialog for the reader, and what he doesn't 9A does behind his back.

**Reply by cdrigby** [reply]
Aha! I didn't realize that, or really, I forgot it. Thanks

**Reply by syvwlch** [reply]
I point this out in the data dictionary but not in the visualization... Clearly I should, and will add a clarification request to the issue tracker.

Thanks!

**Comment by syvwlch** [comment]
Added clarification request 

https://github.com/syvwlch/Data-Ignota/issues/38
